db/payment_icons.yml

```diff
@@ -1562,7 +1562,11 @@
   name: sveaCreditAccount
   label: Svea Credit account
   group: other
+-
+  name: rcbc_direct_debit
```

Comment:
@andykim we're avoiding names and file names with any special characters - can you remove the `_` character in both the name attribute and on the svg file name?